Add rake, cap and shipit tasks to clear memcached cache

sonalkr132 opened the pull request:
Hope it works 🙆

dwradcliffe commented on config/deploy.rb:
```diff
@@ -48,3 +48,14 @@
+namespace :memcached do
+  desc "Flushes memcached instance cache"
+  task :flush do
+    on roles(:app) do
```
We only need to execute this on one of the app servers.

sonalkr132 replied:
So I should just remove that line?

dwradcliffe replied:
You can select just one using primary() I think
http://capistranorb.com/documentation/advanced-features/property-filtering/

sonalkr132 replied:
Among app01..05 which one is primary?

dwradcliffe commented:
Can you set app01.staging as primary too?

dwradcliffe commented:
Looking good, I'll test this before merge.

dwradcliffe commented:
This was almost correct. Needed to add https://github.com/rubygems/rubygems.org/commit/5113297555c98008e36b469239a1dfbb854b003a

sonalkr132 commented:
Thanks. I had added it https://gist.github.com/sonalkr132/e929593f6ddc9e9243af04616b617890 but in the end I removed it because of this: https://www.phusionpassenger.com/library/deploy/apache/automating_app_updates/ruby/#anatomy-1
